src/lib/orm/metadata.go

```diff
@@ -30,33 +30,74 @@ var (
 	cache = sync.Map{}
 )
 
-type key struct {
+// Key ...
+type Key struct {
 	Name       string
 	Filterable bool
 	FilterFunc func(context.Context, orm.QuerySeter, string, interface{}) orm.QuerySeter
 	Sortable   bool
 }
 
-type metadata struct {
-	Keys         map[string]*key
-	DefaultSorts []*q.Sort
+// Column the columns of the model
+type Column struct {
```

Comment:
Seems the `Column` is dup of `Key`?

Reply:
`selector` wants to know the all columns in the table for the model, it's not possible by the Keys because there are `FieldName`, `FilterByXxx` in the Keys.

Reply:
`Key` was removed, now the `Column` only be the fields info from the model.